Chart.yaml

```
@@ -3,7 +3,7 @@ appVersion: 1.5.5
 ossVersion: 1.5.5
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 6.4.8
+version: 6.4.9
```

Please do not change the new version number. We will change it on a new release.
```suggestion
version: 6.4.8
```

---

I have removed the extra commit, which increased the version